sandbox/src/ActivityMapIndex.js

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect } from "react";
+import { Link } from "react-router-dom";
+import ContentSecurityPolicy from "./components/ContentSecurityPolicy";
+import useSendPageViewEvent from "./useSendPageViewEvent";
+
+function ActivityMapIndex() {
+  useSendPageViewEvent();
+
+  useEffect(() => {
+    let buttonClickCounter = 0;
+    const changeButtonText = () => {
+      buttonClickCounter++;
+      let button = document.getElementById("counting-button");
+      button.innerHTML = `Clicked ${buttonClickCounter} times`;
+    };
+
+    const navigateToAdobe = () => {
+      document.location = "https://adobe.com";
+    };
+
+    const sendAPageView = () => {
+      alloy("sendEvent", {
+        xdm: {
+          eventType: "web.webpagedetails.pageViews",
+          web: {
+            webPageDetails: {
+              name: "Page View",
+              pageViews: {
+                value: 1,
+              },
+            },
+          },
+        },
+      });
+    };
+
+    document
+      .getElementById("counting-button")
+      .addEventListener("click", changeButtonText);
+    document
+      .getElementById("send-page-view")
+      .addEventListener("click", sendAPageView);
+    document
+      .getElementById("navigate-to-adobe")
+      .addEventListener("click", navigateToAdobe);
+
+    return () => {
+      document
+        .getElementById("counting-button")
+        .removeEventListener("click", changeButtonText);
+      document
+        .getElementById("send-page-view")
+        .removeEventListener("click", sendAPageView);
+      document
+        .getElementById("navigate-to-adobe")
+        .removeEventListener("click", navigateToAdobe);
+    };
+  }, []);
+
+  return (
```

Comment:
It doesn't look like there's a page view on these pages which will limit the ActivityMap features (which picks up page-name etc.).